Tools/scripts/build_options.py

```diff
@@ -151,6 +151,7 @@ def config_option(self):
 
     Feature('Camera', 'RUNCAM', 'AP_CAMERA_RUNCAM_ENABLED', 'Enable RunCam control', 0, 'Camera'),
 
+    Feature('Copter', 'COPTER_MODE_AVOIDADSB', 'AP_COPTER_MODE_AVOIDADSB_ENABLED', 'Enable AvoidADSB', 0, 'ADSB'),
```

Comment:
I think "COPTER_MODE_AVOIDADSB" should be just "MODE_AVOIDADSB"

Reply:
Hi @peterbarker,

For copter and maybe rover, there are 3 different avoidance features:

1. adsb avoidance (which is silly and should be removed or be combined with path planning as PeterH has recently done for AIS)
2. simple avoidance (e.g. stopping)
3. path planning (dijkstras, bendy ruler)

So I think we need these defines:

a) ADSB_ENABLE which just enables/disables the ADSB sensors
b) MODE_ADSBAVOID_ENABLE which should definitely default to being off and we can move towards removing the mode completely
c) simple avoidance
d) path planning avoidance